doc/yay.8

```diff
@@ -201,22 +201,22 @@ Sort AUR results by a specific field during search\&.
 \fB\-\-answerclean <All|None|Installed|NotInstalled|...>\fR
 .RS 4
 Set a predetermined answer for the clean build menu question\&. This answer
-will be used instead of reading from standard input but will be treaded exactly
+will be used instead of reading from standard input but will be treated exactly
 the same when parsed\&.
 .RE
 .PP
 \fB\-\-answeredit <All|None|Installed|NotInstalled|...>\fR
 .RS 4
 Set a predetermined answer for the edit pkgbuild menu question\&. This answer
-will be used instead of reading from standard input but will be treaded exactly
+will be used instead of reading from standard input but will be treated exactly
 the same when parsed\&.
 .RE
 .PP
 \fB\-\-answerupgrade\fR <Repo|^Repo|None|...>
 .RS 4
-Set a predetermined answer for the upgrade menu question\&. This answer
-will be used instead of reading from standard input but will be treaded exactly
-the same\&.
+Set a predetermined answer for the upgrade menu question\&. Selects which packages
+groups or repos to select for updades\&. This answer will be used instead of
```

Comment:
This answers what packages to not upgrade. Also you can't enter groups unless you mean `1-3 `which I would call ranges to not confuse with groups like `base` and `base-devel`.

Reply:
Ah, apologies- got the logic back-to-front. Updated commit incoming.

Reply:
There's still the bit about groups, I would either leave it out or change it to package ranges.

Reply:
Looks good now thanks.